test/onnx/test_operators.py

```diff
@@ -534,6 +534,12 @@ def test_nonzero(self):
         x = torch.tensor([[[2., 2.], [1., 0.]], [[0., 0.], [1., 1.]]], requires_grad=True)
         self.assertONNX(lambda x: torch.nonzero(x), x)
 
+    def test_stable_opset(self):
```

Comment:
Shall we add some test here to see if opset_version is really enabled? Like, try to construct a situation that opset_version = 9 or 10 has different result?

Reply:
Yeah, I manually did that for opset 10 locally. Since I am about to bump the default one to 10, so i still use 9 here. :-)